### PR TITLE
Add Search Bar

### PR DESCRIPTION
**Pull request by darkboywonder**
This pull request adds a geocoder to the top left of the map which allows users to search by location even down to a specific address. I also took a step towards using a specific map for this application. I plan on adjusting the color in the near future.

screen shot 2018-07-13 at 12 40 35 pm

### CODE REVIEW

File: resources/views/map.blade.php [hunk 1]
```diff
@@ -7,20 +7,42 @@
 
         <script src='https://api.mapbox.com/mapbox-gl-js/v0.44.2/mapbox-gl.js'></script>
         <link href='https://api.mapbox.com/mapbox-gl-js/v0.44.2/mapbox-gl.css' rel='stylesheet' />
+        <link rel='stylesheet' href='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v2.2.0/mapbox-gl-geocoder.css' type='text/css' />
```
Comment (mattstauffer) [inline resources/views/map.blade.php:10]: One thing that doesn't matter--but helps us develop the right mindset--is to pay attention to details. 

Note in this diff we have mixed spacing after the colons in the CSS, mixed single- and double-quotes, some inline styles and some external styles. I'm sure this was a bit of a crazy mental process, but it helps to just develop the attention to detail that gets our brains in a better place to pay attention to detail in bigger places.

**Comment by mattstauffer** [comment]
Looks good dude!

**Comment by darkboywonder** [comment]
Thank you for the notes! I was being quick about it but will for sure continue to increase my awareness going forward.

**Comment by mattstauffer** [comment]
(FYI, it's your project--you do whatever you want! just a note as someone who wants to see you grow :D )

**Pull request merged**